lib/AST/ASTImporter.cpp

```diff
@@ -256,7 +256,6 @@ namespace clang {
       ToD = CreateFun(std::forward<Args>(args)...);
       // Keep track of imported Decls.
       Importer.MapImported(FromD, ToD);
-      Importer.AddToLookupTable(ToD);
```

Comment:
Is this change correct here?

Reply:
You are right, that should not be there. I just have updated.